src/core/usecase.py

```
@@ -10,6 +10,7 @@
 from port.output.visit_repository import VisitRepository
 from infrastructure.env_variable import AUTHENTICATION_CODE
 import hmac
+from infrastructure.log.logger import logger
```

Ruff (0.8.2)

`infrastructure.log.logger.logger` imported but unused

Remove unused import: `infrastructure.log.logger.logger`

(F401)